### PR TITLE
Add more options for upload translations

### PR DESCRIPTION
**Pull request by Bartlebyy**
Add new variables for upload translations

image
from https://support.crowdin.com/configuration-file/

### CODE REVIEW

**Comment by andrii-bodnar** [comment]
Hi @Bartlebyy , thank you for the contribution!

File: entrypoint.sh [hunk 1]
```diff
@@ -64,6 +64,18 @@ upload_sources() {
 }
 
 upload_translations() {
+  if [[ -n "$INPUT_AUTO_APPROVE_IMPORTED" ]]; then
```
Comment (VBeytok) [inline entrypoint.sh:67]:
```suggestion
  if [[ "$INPUT_AUTO_APPROVE_IMPORTED" = true ]]; then
```

File: entrypoint.sh [hunk 2]
```diff
@@ -64,6 +64,18 @@ upload_sources() {
...
+    OPTIONS="${OPTIONS} --auto-approve-imported"
+  fi
+
+  if [[ -n "$INPUT_IMPORT_EQ_SUGGESTIONS" ]]; then
```
Comment (VBeytok) [inline entrypoint.sh:71]:
```suggestion
  if [[ "$INPUT_IMPORT_EQ_SUGGESTIONS" = true ]]; then
```

File: entrypoint.sh [hunk 3]
```diff
@@ -64,6 +64,18 @@ upload_sources() {
...
+    OPTIONS="${OPTIONS} --import-eq-suggestions"
+  fi
+
+  if [[ -n "$INPUT_IMPORT_DUPLICATES" ]]; then
```
Comment (VBeytok) [inline entrypoint.sh:75]:
```suggestion
  if [[ "$INPUT_IMPORT_DUPLICATES" = true ]]; then
```

**Pull request merged**